src/home_work2/task1.js

```diff
@@ -0,0 +1,76 @@
+import * as Joi from '@hapi/joi';
+const express = require('express');
+const collection = require('./db');
+const bodyQuerySchema = require('./validations');
+const app = express();
+
+const bodyParser = require('body-parser');
+app.use(bodyParser.urlencoded({ extended: true }))
+app.use(bodyParser.json());
+
+const port = process.env.PORT || 8080;
+const router = express.Router();
+
+const validator = require('express-joi-validation').createValidator({})
```

Comment:
Same as above - by mixing your code and imports/exports you're making it harder to read.
Some default linting setting will even through errors at it.